src/controllers/phone.controllers.ts

```diff
@@ -99,10 +99,23 @@ const getRecommended = async (req: Request, res: Response) => {
   res.send(getRecommendedPhones);
 };
 
+const getPhones = async (req: Request, res: Response) => {
+  const phoneName = req.params['name'];
+
+  const getPhoneByName = await sequelize.query(
+    `SELECT * FROM "PhoneDetails" WHERE "name"='${phoneName}'`, {
```

Comment:
Im not sure how exactly we want it to work, but I think the search bar will be searching for products on PhonesPage,  so we want to SELECT * FROM "Phones".
Also I don't think this condition WHERE is correct, because someone would have to type the exact name of the product. So I think we want to use WHERE "name" LIKE ...